### DISCUSSION TITLE
Help me!Description show unknown

### DISCUSSION THREAD

**Discussion by youland** [photoprism/photoprism#4231] [General]
![Snipaste_2024-05-04_11-08-29](https://github.com/photoprism/photoprism/assets/3339858/440bec2c-c46d-4aea-a6c6-c8fdeeef3976)
![Snipaste_2024-05-04_11-08-18](https://github.com/photoprism/photoprism/assets/3339858/fbb6254d-74b5-4b0a-a0d1-a9792f1b643e)
![Snipaste_2024-05-04_11-12-18](https://github.com/photoprism/photoprism/assets/3339858/42b49b4a-4554-4da0-97c6-f6df7609c5aa)

How to remove these messy code in description,thanks.

Installed on docker.

**Comment by graciousgrey** [comment]
PhotoPrism is reading metadata from your original files.

https://docs.photoprism.app/user-guide/library/metadata/
https://www.photoprism.app/kb/metadata

You can check the metadata of your file by running `exiftool -j filename.mp4` in a terminal.

To delete a description you can use the [edit dialog](https://docs.photoprism.app/user-guide/organize/edit/), however this will not (yet) modify the metadata of your original file.
